core/src/main/resources/error/error-classes.json

```diff
@@ -14,13 +21,20 @@
     "message" : [ "Found duplicate keys '%s'" ],
     "sqlState" : "23000"
   },
+  "EXCEED_ARRAY_SIZE_WHEN_ZIP_MAP" : {
```

Comment:
This doesn't make much sense grammatically. How about `ZIPPED_MAPS_EXCEED_SIZE_LIMIT`?